scripts/quests/bastok/Beauty_And_The_Galka.lua

```diff
@@ -80,7 +80,10 @@ quest.sections =
                 onTrigger = quest:event(4),
 
                 onTrade = function(player, npc, trade)
-                    if npcUtil.tradeHasExactly(trade, { { xi.items.CHUNK_OF_ZINC_ORE, 1 } }) then
+                    if
+                        npcUtil.tradeHasExactly(trade, { { xi.items.CHUNK_OF_ZINC_ORE, 1 } }) and
+                        not player:hasKeyItem(xi.ki.PALBOROUGH_MINES_LOGS)
+                        then
```

Comment:
Nit but the `then` should be bumped back by 1 indent

Reply:
updated.